kura/examples/org.eclipse.kura.example.eddystone.scanner/OSGI-INF/metatype/org.eclipse.kura.example.eddystone.scanner.EddystoneScannerExample.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<MetaData xmlns="http://www.osgi.org/xmlns/metatype/v1.2.0" localization="en_us">
+    <OCD id="org.eclipse.kura.example.eddystone.scanner.EddystoneScannerExample"
+         name="Eddystone Scanner" 
+         description="Eddystone Scanning Example.">
+         
+         <Icon resource="http://s3.amazonaws.com/kura-resources/application/icon/eddystone_logo.png" size="32"/>
+         
+         <AD id="enableScanning"
```

Comment:
We use different conventions on the parameter names. Probably we should chose one and modify accordingly all the bundles. Should I use this.is.a.property convention?